README.md

```diff
@@ -81,15 +76,15 @@ alias(
   actual = "@maven//:junit_junit_4_12",
 )
 
-java_import(
+jvm_import(
```

Comment:
While you're here, if you want, these are actually the versionless targets now and the alias has the version on line 74. But if not, lgtm and I can do it in a follow up

Reply:
Done